cloud/ec2_fleet.go

```diff
@@ -437,6 +437,11 @@ func (m *ec2FleetManager) requestFleet(ctx context.Context, ec2Settings *EC2Prov
 	}
 	err = validateEc2CreateFleetResponse(createFleetResponse)
 	if err != nil {
+		grip.Error(message.WrapError(err, message.Fields{
```

Comment:
I think this will end point up the address of the pointers inside the structs

Reply:
I've found it worked well in other places, such as https://github.com/evergreen-ci/evergreen/blob/c8c0d8a3a82e147624b219341750f4ca065f8534/cloud/ec2_client.go#L168 though I never dug into the mechanics